charts/seed-bootstrap/charts/elastic-kibana-curator/values.yaml

```diff
@@ -53,6 +56,59 @@ elasticsearch:
         perObject: 89
         weight: 5
         unit: Mi
+  basicAuth: YWRtaW46YWRtaW4K
```

Comment:
hardcoded password?

Reply:
NO, this is just an example how does basicAuth looks. In this example it is admin:admin. It is not a real password. The controller manager will override its value during the runtime.